README.md

```diff
@@ -191,6 +191,19 @@ An integer indicating the number of events (spans, transactions, or errors)
 sent by the client. An event is considered sent when the HTTP request
 used to transmit it have ended.
 
+### `client.config(options)`
+
+Update the configuration given to the `Client` constructor. All
+configuration options can be updated except:
+
+- The protocol part of the `serverUrl` (`http` vs `https`)
```

Comment:
Can we make protocol work too? Without that, it's basically impossible to late-configure it to use https, because the default http serverUrl will get used initially, locking that down. 🤔

Reply:
I looked into that when I made the PR, but it's much more complicated. So I ended up concluding that it would be simpler to land this PR as is, which would allow for late setting of anything inside metadata (including framework). Then we can make a separate PR to allow delayed start, which would solve this.